app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -363,6 +378,13 @@ data class User private constructor(
                     }
                     user.likes = newLikes
                 }
+                DISLIKES -> {
```

Comment:
I don't think I missed them, but I think you forgot to add this in the UserUnitTest

Reply:
Yes you are right! I've added them.